DZ1_var11.py

```diff
@@ -0,0 +1,95 @@
+# Вариант 11
```

Comment:
extremely hard too check a logic of your program. let's make a call and discuss the code